tests/functional/ctu/ctu_failure/test_ctu_failure.py

```diff
@@ -149,21 +206,27 @@ def __do_ctu_all(self, reparse, extra_args=None):
         if extra_args is not None:
             cmd.extend(extra_args)
         cmd.append(self.buildlog)
-        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT,
-                                         cwd=self.test_dir, env=self.env)
-        return output
 
-    def __do_ctu_analyze(self, reparse):
-        """ Execute CTU analyze phase. """
-
-        cmd = [self._codechecker_cmd, 'analyze', '-o', self.report_dir,
-               '--analyzers', 'clangsa', '--ctu-analyze']
-        if reparse:
-            cmd.append('--ctu-on-the-fly')
-        cmd.append(self.buildlog)
-        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT,
-                                         cwd=self.test_dir, env=self.env)
-        return output
+        def show(out, err):
```

Comment:
You could put this under `libtest` it might be useful for other tests.

Reply:
:+1: